crates/ln-dlc-node/src/tests/add_dlc.rs

```diff
@@ -23,18 +23,24 @@ async fn given_lightning_channel_then_can_add_dlc_channel() {
     app.keep_connected(coordinator.info).await.unwrap();
 
     app.fund(Amount::from_btc(0.1).unwrap()).await.unwrap();
-
-    let channel_details = app
-        .open_channel(coordinator.info, 50000, 50000)
+    coordinator
+        .fund(Amount::from_btc(0.1).unwrap())
         .await
         .unwrap();
 
+    coordinator.open_channel(&app, 50000, 50000).await.unwrap();
+    let channel_details = app.channel_manager.list_usable_channels();
+    let channel_details = channel_details
+        .iter()
+        .find(|c| c.counterparty.node_id == coordinator.info.pubkey)
+        .unwrap();
```

Comment:
https://github.com/get10101/10101/pull/169/commits/dc8c5e2b43923035927af3e2710f5e9e821052c9

Reply:
Since I turned the initiator of the channel around I can not use the channel_details returned by the coordinator. Hence I am looking for the channel details in the app channel_manager.